labcontrol/gui/js_tests/tests.js

```diff
@@ -0,0 +1,7 @@
+QUnit.test( "hello test", function( assert ) {
```

Comment:
Same comment here as for index.html

Reply:
I suppose my thinking was to submit these files so they're in the repo and we have a complete, if trivial working framework. I can drop them from the PR however.

Reply:
Ok, I guess we can leave them in ... but when I put in my PR that involves some real LabControl tests, I assume I'm taking them right back out, yes?